linode/vpcsubnet/resource_test.go

```diff
@@ -0,0 +1,206 @@
+//go:build integration
+
+package vpcsubnet_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/linode/linodego"
+	"github.com/linode/terraform-provider-linode/linode/acceptance"
+	"github.com/linode/terraform-provider-linode/linode/helper"
+	"github.com/linode/terraform-provider-linode/linode/vpcsubnet/tmpl"
+)
+
+var vpcID int
+
+func init() {
+	resource.AddTestSweepers("linode_vpc_subnet", &resource.Sweeper{
+		Name: "linode_vpc_subnet",
+		F:    sweep,
+	})
+
+	client, err := acceptance.GetTestClient()
+	if err != nil {
+		log.Fatal(fmt.Errorf("Error getting client: %s", err))
+	}
+
+	vpc, err := client.CreateVPC(context.Background(), linodego.VPCCreateOptions{
+		Label:  acctest.RandomWithPrefix("tf-test"),
+		Region: acceptance.GetRandomRegionWithCaps([]string{"VPCs"}),
+	})
```

Comment:
`GetRandomRegionWithCaps` returns the region and an error here we'll need to handle the error accordingly.
```suggestion
	region, err := acceptance.GetRandomRegionWithCaps([]string{"VPCs"})
	if err != nil {
		log.Fatalf("Failed to get test region: %s", region)
	}

	vpc, err := client.CreateVPC(context.Background(), linodego.VPCCreateOptions{
		Label:  acctest.RandomWithPrefix("tf-test"),
		Region: region,
	})
```

Reply:
@zliang-akamai yes, also fixed in this PR.